api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -11096,6 +11260,49 @@ public CohortDefinition getPatientsOnMQCat18Numerator() {
     return cd;
   }
 
+  public CohortDefinition getpregnantWithCargaViralHigherThan50() {
+    CompositionCohortDefinition compositionCohortDefinition = new CompositionCohortDefinition();
+
+    compositionCohortDefinition.addParameter(new Parameter("startDate", "startDate", Date.class));
+    compositionCohortDefinition.addParameter(new Parameter("endDate", "endDate", Date.class));
+    compositionCohortDefinition.addParameter(
+        new Parameter("revisionEndDate", "revisionEndDate", Date.class));
+    compositionCohortDefinition.addParameter(new Parameter("location", "location", Location.class));
+
+    compositionCohortDefinition.setName(
+        "Cat 13 - DEN 18: # de MG na 1ª linha de TARV com registo de resultado de CV");
+
+    CohortDefinition pregnantClinicalWithCargaViralHigherThan50 =
+        QualityImprovement2020Queries.getMQ13DenB4_Clinica_CV_50(
+            hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId(),
+            hivMetadata.getHivViralLoadConcept().getConceptId(),
+            hivMetadata.getYesConcept().getConceptId(),
+            commonMetadata.getPregnantConcept().getConceptId());
```

Comment:
Please use the recommended query.
QualityImprovement2020Queries.getMQ13DenB5_P4()

Reply:
Done